examples/get-started/flutter-for/ios_devs/lib/list.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(
+    const App(),
+  );
+}
+
+class App extends StatelessWidget {
+  const App({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      home: HomePage(),
+    );
+  }
+}
+
+// #docregion SimpleList
+// create a class that holds each person's data
```

Comment:
Would consider removing - doesn't add much info

Reply:
Removing the comment on line 24?